src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -539,13 +459,13 @@ void DefaultPolicy::NoteBool(InlineObservation obs, bool value)
 
 #if defined(DEBUG) || defined(INLINE_DATA)
 //------------------------------------------------------------------------
-// DumpXml: Dump DefaultPolicy data as XML
+// DumpXml: Dump ExtendedDefaultPolicy data as XML
```

Comment:
Does `DefaultPolicy` no longer override `DumpXml`? Seems like it should, and the extended policy dump should layer on top of it (or something).